### ISSUE TITLE
Trouble to update this mod to 1.12.2

### ISSUE THREAD

**Issue by 76935** [CallMeFoxie/BetterSleeping2#13]
Trouble to update this mod to 1.12.2 I really like this mod

**Comment by CallMeFoxie** [comment]
Yes. Making it compile is easy. Making it work is much harder and I haven't had the opportunity to make it work past 1.10 sadly. Maybe with 1.14 or 1.15? :thinking:
